remote_nodes.sh

```diff
@@ -91,13 +93,24 @@ function setup_remote_cluster() {
 	sudo $IPTABLES -I FORWARD 1 -o ${REMOTE_CLUSTER_NAME} -i ${BAREMETAL_NETWORK_NAME} -j ACCEPT
 
 	# Create the NS and apply the manifests
-	oc create ns ${NS}
-	oc apply -f ${OCP_DIR}/remote_host_manifests.yaml
+	if [[ $APPLY_REMOTE_NODES = "true" ]]
+	then
+		# OCP should have already created openshift-machine-api namespace
+		if [[ $NS != "openshift-machine-api"  ]]
+		then
+			oc create ns ${NS}
+		fi
+		oc apply -f ${OCP_DIR}/remote_host_manifests.yaml
+	fi
+
 }
 
 function cleanup_remote_cluster() {
-	# Remove ns
-	oc delete ns ${NS}
+	# Remove namespace - OCP will not allow you to remove openshift-machine-api
+	if [[ $NS != "openshift-machine-api" ]]
```

Comment:
For the sake of overengineering and future prediction, what do you think about having a list of namespaces (or space-separated string of namespaces) that should be "protected"?

Reply:
Doing some reason on this.. we might be able to just if the namespace starts with "openshift" but having a running list is good too.

Reply:
Either works for me! Considering all the `openshift-` namespaces as "protected" sounds sane.